### ISSUE TITLE
iOS App using SDK 1.1.0 and "IMPL_APP_CONTROLLER_SUBCLASS" will not launch.

### ISSUE THREAD

**Issue by T264** [firebase/quickstart-unity#30]
In order to extend UnityAppController of iOS App, we write as follows.
```
#import "UnityAppController.h"

@interface MyAppController : UnityAppController
@end

@implementation MyAppController

@end

IMPL_APP_CONTROLLER_SUBCLASS(MyAppController)
```
However, when I use Firebase SDK 1.1.0, the launched application will stop with a black screen.
Use SDK 1.0.0, the iOS App using "IMPL_APP_CONTROLLER_SUBCLASS" also works fine.

Is this problem known?

**Comment by stewartmiles** [comment]
Which components of the SDK are you using?  Does this happen when just importing FirebaseAnalytics for example?

**Comment by T264** [comment]
@stewartmiles 

Hello, stewartmiles.

I using FirebaseMessaging.
So, It occurred when adding the "MyAppController" to "quickstart-unity/messaging".

Thanks,

**Comment by stewartmiles** [comment]
This should be resolved in version 1.1.1
https://firebase.google.com/support/release-notes/unity#1.1.1
